### ISSUE TITLE
GETTING "INTERNAL SERVER ERROR 500" While User Registration & Profile Update

### ISSUE THREAD

**Issue by hrithikram15** [meabhisingh/mernProjectEcommerce#30]
Hello.. I am facing an issue while working on User Registration & Profile Update. I am getting JS alert as "INTERNAL SERVER ERROR" as i click on REGISTER & UPDATE PROFILE buttons.

Can someone help me with this... @meabhisingh 
![Screenshot (123)](https://user-images.githubusercontent.com/101961984/189475198-8070901b-6f2b-4c1a-a6d6-d4f131ef6a03.png)

![Screenshot (122)](https://user-images.githubusercontent.com/101961984/189475180-28093ff0-6526-45fc-bec1-8b5beda525b4.png)

**Comment by sayedulkrm** [comment]
Same problem bro.
Did you find any solution?

**Comment by hrithikram15** [comment]
> Same problem bro. Did you find any solution?

Yep...  In my case it is bcoz of Changing Internet/WiFi connection which led to change in the Proxy address that we have mentioned in "package.json" while setting up the project. So, fix you proxy address according to ur current working netwotk connection.

If its not the proxy issue(means if u have not changes ur Internet connection) then, this SERVER ERROR is raised because we have not selected any image file separately for Updating Profile 0r During Registration. Its because as per the code we used in this project, we cannot skip uploading image wherever it is asked to upload in cloudinary. So, please check once by choosing file separately for uploading. 
As u can see in the image i have uploaded in above section(UPDATE PROFILE) i have not selected any file for Profile image during Profile update & i just simply left the same img which was used before during profile registration. Hence, this throws a INTERNAL SERVER ERROR, then i just selected file once again from the system & uploaded.

Hope this works for u...

**Comment by anjali9294** [comment]
can any one help me with loadUser whenever i refreash the page me react component state is gone i used storedispatch(loadUser()) in app.js but again loagin again error occur again and again

**Comment by sayedulkrm** [comment]
@anjali9294 Visit my profile. Hopefully, you will find a solution om your code, and please give it a start. 
Thank You.

**Comment by anjali9294** [comment]
@sayedulkrm  actually my  cookie token is not storing in browser but its stored in postman and thats why my loadUser() give me error again and again so i don't know how to solve it

**Comment by sayedulkrm** [comment]
@anjali9294 did you check my code? Check the spelling or restart the browser or server again. Because I didn't find any problem storing cookies.

**Comment by anjali9294** [comment]
@sayedulkrm  i check the code of backend as well as front end but no mistake in code my cookie is generated in postman n all backend api's working fine but not saving cookie in frontend bcoz of that cookie token i cant  logged and loadUser give me an error of unauthorized user bcoz there is no token

**Comment by sayedulkrm** [comment]
@anjali9294  You are using postman, right? Did you download Postman Agent?

**Comment by anjali9294** [comment]
@sayedulkrm  i have postman app not agent

**Comment by sayedulkrm** [comment]
@anjali9294 download Postman Agent desktop software. Hope that your problem will be fixed.
And try to use it from the browser.

**Comment by anjali9294** [comment]
@sayedulkrm ok but i don't think so it will work

**Comment by sayedulkrm** [comment]
@anjali9294 it will work I think. Because I think cookie saves in the browser. So try to use postman.com from the Chrome browser.

**Comment by anjali9294** [comment]
@sayedulkrm i used cookie.set method it set the cookie but when it comes to loadUser method it still not woking its giving me login error again

**Comment by anjali9294** [comment]
@sayedulkrm  hi can you check my code ?

**Comment by sayedulkrm** [comment]
@anjali9294 okay sure

**Comment by anjali9294** [comment]
@sayedulkrm can we connect on discord

**Comment by anjali9294** [comment]
@sayedulkrm my discord id is   anjaliparihar#4966     and telegram is          anjali9294

**Comment by sayedulkrm** [comment]
check

**Comment by anjali9294** [comment]
@sayedulkrm where ?
